### ISSUE TITLE
Python does not build  (OS X)

### ISSUE THREAD

**Issue by jon-jacky** [rumpkernel/rumprun-packages#41]
I am trying to build rumprun-packages/python on Mac OS X 10.11.1.

On Oct 29 I successfully built and ran (on qemu) the helloer.c rumprun
unikernel example on this Mac, following the directions in the wiki.

This happens when I follow the directions on the
rumprun-packages/python/README.md:

$ pwd
/Users/jon/rumprun-packages/python

$ make
(cd build; LDFLAGS="-static -static-libgcc" CPPFLAGS=" -I/Users/jon/rumprun-packages/pkgs/include -static" CFLAGS=" -static" CC=x86_64-rumprun-netbsd-gcc CONFIG_SITE=config.site ./configure --prefix=/Users/jon/rumprun-packages/python/build/pythondist --disable-shared --host=x86_64-rumprun-netbsd --build x86_64--netbsd --disable-ipv6 --without-ensurepip)
configure: loading site script ./config.site
checking build system type... x86_64--netbsd
checking host system type... x86_64-rumprun-netbsd
checking for python interpreter for cross build... python3.4
checking for --enable-universalsdk... no
checking for --with-universal-archs... no
checking MACHDEP... configure: error: cross build not supported for x86_64-rumprun-netbsd
make: **\* [build/Makefile] Error 1

**Comment by rday** [comment]
I just rebuilt from scratch, but I'm not able to simulate the error on OSX
10.10.5.

Could you try running the make with the MACHDEP env var set?

mbp$ MACHDEP=netbsd make

**Comment by jon-jacky** [comment]
I tried 
$ MACHDEP=netbsd make
...
checking MACHDEP... configure: error: cross build not supported for x86_64-rumprun-netbsd
make: **\* [build/Makefile] Error 1

It failed in exactly the same way.

**Comment by rday** [comment]
Did you get any errors while the patches were being applied? Line 3348
should be detecting that host.

```
      *-*-netbsd*)
              _host_cpu=$host_cpu
```

If I reverse my patches, I get the same error you do.

**Comment by jon-jacky** [comment]
> Did you get any errors while the patches were being applied? 

Yes!   I got my installation into a confused state after I noticed the rumprun-packages/python repo was updated yesterday after I tried make for the first time.    I cleared the problem by cloning a fresh copy of the rumprun-packages repo this morning.  Now it looks like the make command completed successfully.   Thanks so much for your prompt and very helpful comments.
